src/tools/perf/ucp_tests.cc

```diff
@@ -44,12 +44,13 @@ class ucp_perf_test_runner {
         iov_length_it = 0;
         for (iov_it = 0; iov_it < iovcnt; ++iov_it) {
             iov[iov_it].buffer = (char *)buffer + iov_length_it;
-            iov[iov_it].length = m_perf.params.msg_size_list[iov_it];
+            iov[iov_it].dt     = ucp_dt_make_contig(m_perf.params.msg_size_list[iov_it]);
```

Comment:
The proposed API change breaks the existing UT (which this line fixes). Renaming length to count still requires a fix in the UT. This is not IOV implementation - the IOV implementation is 1K+ LOC. Are you sure you want to split it into two PRs, one right after the other?